File: MRSource/Framework/DBTimeLine/Framework.DBTimeLine/DBModulePersister.vb
Imports MRFramework

Public Class DBModulePersister
    Inherits MRPersisting.MRPersister

    Public Overrides ReadOnly Property DataBaseTableName As String
        Get
            Return "DBTimeLine.Module"
        End Get
    End Property
    Public Overrides ReadOnly Property SQL As String
        Get
            Return "SELECT ID, ClassName, AssemblyName, DefaultSchemaName, Active FROM " & DataBaseTableName
        End Get
    End Property
End Class

File: MRSource/DBTimeLiners/DBTimeLiners.DBModules/Persisters/DBModulePersister.cs
namespace DBTimeLiners.DBModules.Persisters
{
    class DBModulePersister : Framework.Persisting.Persister
    {
        protected override string DataBaseTableName
        {
            get
            {
                return "DBTimeLine.Module";
            }
        }

        protected override string Sql
        {
            get
            {
                return "SELECT * FROM " + DataBaseTableName;
            }
        }
    }
}
